### DISCUSSION TITLE
New button numbering with UDP control

### DISCUSSION THREAD

**Discussion by dubbledex** [bitfocus/companion#3133] [Q&A] [answered]
Hi all. 
I am using an extron panel and the unidirectional UDP driver. To change my bitfocus buttons it was really easy, I had the ip and port, and just put in (for example) "BANK-PRESS 2 15" and it would trigger button 15 on bank 2. 
I updated companion and I see the button numbers have changed - and I read up for the reason of this, sounded sensible. 
My current send code still works from the extron device, however I want to move things around. 
How to I trigger the buttons using the new format? I tried "BANK-PRESS 2 0 0" and it didn't switch anything. 
Thanks!

**Comment by Julusian** [comment, marked as answer]
The settings tab describes the protocols.
For UDP:
![image](https://github.com/user-attachments/assets/8e9a025d-a257-492c-b84d-e087e2c8d12e)

Which for your example would be something like `LOCATION 2/0/0 PRESS`

**Reply by dubbledex** [reply]
Thank you! Worked like a dream
